src/components/structures/TimelinePanel.tsx

```diff
@@ -352,7 +353,7 @@ class TimelinePanel extends React.Component<IProps, IState> {
         if (differentEventId || differentHighlightedEventId || differentAvoidJump) {
             logger.log(
                 `TimelinePanel switching to eventId ${this.props.eventId} (was ${prevProps.eventId}), ` +
-                    `scrollIntoView: ${this.props.eventScrollIntoView} (was ${prevProps.eventScrollIntoView})`,
+                `scrollIntoView: ${this.props.eventScrollIntoView} (was ${prevProps.eventScrollIntoView})`,
```

Comment:
Normally, the code is already formatted by Prettier. Please do not change the formatting :)

Reply:
Oh, Okay. I see, So basically normal `join` event is needed. I got that. Yes, I won't change the formatting as it was automatically done by the package installed in my vs code. Thanks! Let me do the changes!